taskcluster/fenix_taskgraph/transforms/build.py

```diff
@@ -50,15 +51,16 @@ def add_shippable_secrets(config, tasks):
                 ('mls', '.mls_token'),
             )])
         else:
-            task["run"]["pre-gradlew"] = [[
-                "echo", '"{}"'.format(fake_value), ">", target_file
-            ] for fake_value, target_file in (
-                ("--", ".adjust_token"),
-                ("", ".digital_asset_links_token"),
-                ("-:-", ".leanplum_token"),
-                ("", ".mls_token"),
+            dummy_secrets.extend([{
+                "content": fake_value,
+                "path": target_file,
+            } for fake_value, target_file in (
+                ("faketoken", ".adjust_token"),
```

Comment:
I checked each real secret to look at what they really look like. The new ones are closer to real token than the former ones.